pybaseball/statcast_pitcher_spin.py

```diff
@@ -0,0 +1,140 @@
+from pybaseball import statcast_pitcher
+import pandas as pd
+import numpy as np
+
+K = .005153 #Environmental Constant
+SIG_DIG = 4
```

Comment:
is rounding really necessary? if so maybe the result should be rounded only in the last step?

Reply:
I had also thought that rounding at each step was superfluous, but I compared my testing data to the excel sheet that I was doing the calculations for and there were marginal errors that compounded as I went further in the calculations and amounted to non-insignificant differences in the output.

If we can verify the accuracy of the functions I built, then we can absolutely remove rounding at every step

Reply:
I would make a suggestion that we leave out the rounding at each step perhaps for now and then ask Alan Nathan nicely on Twitter if he can check if the math is accurate?

Reply:
I think that;s unnecessary. I confirmed that this code agrees with Alan Nathan's spreadsheet, not just the end result but each individual cell.

Reply:
Bumping this again -- @bdilday -- should we round intermediate steps or only at the end in the final formatted df?

Reply:
OK yeah then nerf the roundign